src/crl.rs

```diff
@@ -23,25 +23,27 @@ pub struct BorrowedCertRevocationList<'a> {
     /// A `SignedData` structure that can be passed to `verify_signed_data`.
     pub(crate) signed_data: signed_data::SignedData<'a>,
 
-    /// Identifies the entity that has signed and issued this
-    /// CRL.
-    pub(crate) issuer: untrusted::Input<'a>,
+    /// A monotonically increasing sequence number for a given CRL scope and CRL issuer.
+    crl_number: Option<&'a [u8]>,
 
     /// Indicates the issue date of this CRL.
-    pub this_update: Time,
+    #[allow(dead_code)] // May be useful in the future.
```

Comment:
Unless you have a concrete use case for the near future in mind, I would probably get rid of the unused time fields here. Not like it's a great burden to add them back if/when we need them, right?

Reply:
Dropped all the unused/unexposed fields.